styles.css

```diff
@@ -372,6 +368,153 @@ section {
   line-height: 16px;
 }
 
+.popup-bg {
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100vw;
+  height: 100%;
+  background-color: rgba(193, 199, 208, 0.9);
+}
+
+.popup {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  align-self: center;
+  height: 95vh;
+  width: 86%;
+  position: fixed;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  background-color: #fff;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+}
+
+.popup-desktop {
+  top: 50%;
+  height: 95%;
+  width: 63%;
+}
+
+.closePopup {
+  position: absolute;
+  left: 86%;
+  top: 36px;
+  width: 12px;
+  height: 12px;
+}
+
+.closePopupDesktop {
+  align-self: flex-end;
+  width: 12px;
+  height: 12px;
+  margin-right: 38px;
+  margin-top: 38px;
+}
+
+.popupImg {
+  margin-top: 16px;
+  left: 27px;
+  top: 16px;
+  width: 95%;
+  height: 220px;
+  align-self: center;
+}
+
+.popupImgDesktop {
+  margin-top: 16px;
+  left: 27px;
+  top: 16px;
+  width: 95%;
+  height: 56%;
+  align-self: center;
+}
+
+.popupTitle {
+  text-align: left;
+  font-family: 'Inter', sans-serif;
+  font-size: 32px;
+  margin-left: 16px;
+}
+
+.popupTitleDesktop {
+  display: flex;
+  width: 95%;
+}
+
+.popupTitleDesk {
+  text-align: left;
+  font-family: 'Inter', sans-serif;
+  font-size: 32px;
+  margin-left: 19px;
+  margin-top: 16px;
+  width: 64%;
+}
```

Comment:
- [ ] [REQUIRED] Good job so far! Please change the `heading` color to correct one => `color: #172B4D;` 

![image](https://github.com/aamir-asaram/Portfolio/assets/63915024/927f05a4-c1f3-411c-aa9d-4de6022dd36d)